README.md

```diff
@@ -840,6 +840,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Vivek Jaiswar](https://github.com/Narcos101)
   - [Vivek Singh](https://github.com/vivmost)
   - [Vladyslav Kyiashko](https://github.com/vlakyi)
+  - [Vishal Rathore](https://github.com/vishalcoder139)
```

Comment:
Please add your name in line 837, after Viraj Shukla.

Reply:
Ok i understand, and i also correct this issue . thank you for your support.

Reply:
yup